src/components/input/range/index.tsx

```diff
@@ -1,22 +1,36 @@
-import { Component, Event, EventEmitter, h, Host, Prop, Watch } from "@stencil/core"
+import { Component, Event, EventEmitter, h, Host, Method, Prop, Watch } from "@stencil/core"
+import { Color } from "../../../model"
+import { Clearable } from "../Clearable"
+import { Input } from "../Input"
+import { Looks } from "../Looks"
 
 @Component({
 	tag: "smoothly-input-range",
 	styleUrl: "style.css",
-	shadow: true,
+	scoped: true,
 })
-export class SmoothlyInputRange {
-	@Prop({ mutable: true }) value = 0
+export class SmoothlyInputRange implements Input, Clearable {
```

Comment:
not strictly necessary but now when we have `componentWillLoad()` we can also implement the interface `ComponentWillLoad`